src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-img {
+    margin: 7px 0;
+    margin-left: 50px;
+    width: 40px;
+    height: 40px;
+  }
+
+  .nav {
```

Comment:
Create a separate file for the nav block 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/fb15669d-ce1f-4a6a-8d41-bb84633f0a9c)